File: pos-backend/src/main/java/com/punto/de/venta/api/rest/modelo/VentasTotalesModelo.java
package com.punto.de.venta.api.rest.modelo;

import java.io.Serializable;
import java.util.List;

import com.punto.de.venta.api.rest.entity.VentaAdicionalEntity;
import com.punto.de.venta.api.rest.entity.VentaTotalEntity;
import com.punto.de.venta.api.rest.entity.VentasEntity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
public class VentasTotalesModelo implements Serializable {
	/**
	 * Serial
	 */
	private static final long serialVersionUID = -4770016295814596927L;

	List<VentasEntity> ventas;

	List<VentaAdicionalEntity> ventasAdicional;

	VentaTotalEntity ventaTotal;

}

File: pos-frontend/src/app/modelo/VentasTotalesModelo.ts
import { VentaAdicional } from './VentaAdicional';
import { Ventas } from './Ventas';
import { VentaTotal } from './VentaTotal';

export interface VentasTotalesModelo {
  ventas: Ventas[];
  ventasAdicional: VentaAdicional[];
  ventaTotal: VentaTotal;
}
